lte/protos/session_manager.proto

```diff
@@ -783,6 +794,23 @@ message SetSMSessionContextAccess {
   RatSpecificContextAccess   rat_specific_context = 2;
 }
 
+
+message UPFSessionState {
+  string seid = 1;            // Session ID
```

Comment:
is this really the session id? or is it the session key?

Reply:
we need to be very explicit here. e.g., if we are passing the tunnel ID or IMSI, etc., let's explicitly define the field for that.

Reply:
@gurrapualt @panyogesh can we change this keyword to what we have finalized ?